README.md

```diff
@@ -8,6 +8,12 @@ This project provides common APIs for applications that live on vets.gov. This r
 
 1. Install Ruby 2.3. (It is suggested to use a Ruby version manager such as [rbenv](https://github.com/rbenv/rbenv#installation) and then to [install Ruby 2.3](https://github.com/rbenv/rbenv#installing-ruby-versions)).
 1. Install Bundler to manage dependencies: `gem install bundler`
+1. Install Postgres (on Mac): `brew install postgres`
+1. Install gem dependencies: `cd vets-api; bundle install`
+
+### Database Setup
+1. Start Postgres: `postgres -D /usr/local/var/postgres`
+1. Create dev database: `bundle exec rake db:create`
```

Comment:
we don't have a schema file yet so that command would fail - when it gets added, the documentation should be updated accordingly :)